app/controllers/bulk_upload_sales_logs_controller.rb

```diff
@@ -14,7 +14,7 @@ def show
   end
 
   def update
-    if form.valid?
+    if form.valid? && form.save!
```

Comment:
Same as above?
Would it make sense to have a parent controller? Seems like a lot of the methods could be shared